apps/condo/domains/organization/hooks/useTableColumns.tsx

```diff
@@ -1,110 +1,127 @@
+import { Typography } from 'antd'
+import { useRouter } from 'next/router'
 import React, { useMemo } from 'react'
 import { identity } from 'lodash/util'
 import get from 'lodash/get'
 
 import { useIntl } from '@condo/next/intl'
 
 import { getFilteredValue } from '@condo/domains/common/utils/helpers'
-import { getTextFilterDropdown, getFilterIcon } from '@condo/domains/common/components/TableFilter'
+import { getFilterIcon } from '@condo/domains/common/components/TableFilter'
 import { getTableCellRenderer } from '@condo/domains/common/components/Table/Renders'
 
-import { createSorterMap, IFilters } from '../utils/helpers'
-import { OrganizationEmployeeRole } from '../utils/clientSchema'
-import { getOptionFilterDropdown } from '../../common/components/Table/Filters'
+import { IFilters } from '@condo/domains/organization/utils/helpers'
+import { OrganizationEmployeeRole } from '@condo/domains/organization/utils/clientSchema'
+import { getOptionFilterDropdown } from '@condo/domains/common/components/Table/Filters'
+import { getFilterDropdownByKey } from '@condo/domains/common/utils/filters.utils'
+import { parseQuery } from '@condo/domains/common/utils/tables.utils'
+import { SpecializationScope } from '@condo/domains/scope/utils/clientSchema'
```

Comment:
it looks like a part of the scope domain

Reply:
probably it's better to move the SpecializationScope to the Organization domain?